extensions/2.0/Khronos/KHR_image_formation/README.md

```diff
@@ -0,0 +1,197 @@
+<!--
+Copyright 2018-2020 The Khronos Group Inc.
+SPDX-License-Identifier: LicenseRef-KhronosSpecCopyright
+-->
+
+# KHR\_image\_formation
+
+## Contributors
+
+* Romain Guy, Google [@romainguy](https://twitter.com/romainguy)
+* Emmett Lalish, Google [@emmettlalish](https://twitter.com/emmettlalish)
+
+Copyright 2018-2020 The Khronos Group Inc. All Rights Reserved. glTF is a trademark of The Khronos Group Inc.
+See [Appendix](#appendix-full-khronos-copyright-statement) for full Khronos Copyright Statement.
+
+## Status
+
+Draft
+
+## Dependencies
+
+Written against the glTF 2.0 spec.
+
+## Overview
+
+The scene referred values computed during the lighting pass using the glTF 2.0 spec are luminance values, expressed in cd.m-2 or nits. It follows that light incident to our virtual optical system can cover a large range of values when physical light units (KHR_lights_punctual) are used, from 10-5 cd.m-2 for starlight to 109cd.m-2 for the sun. However the common methods we have of storing and displaying images have limited dynamic range. Of course this problem does not just apply to rendering; the human eye also has limited dynamic range, and of course photographs printed on paper (or displayed on a screen) cannot possibly reproduce the range  of intensity that was incident on the lens. 
+
+This extension is intended to allow the producer of a glTF to define all the parameters necessary for a consumer (renderer) to generate a consistent output image of the scene, given consistent lighting and view. Consistent does not imply pixel-identical, as the rendering is still subject to many arbitrary approximations, but simply that the output would be identical if the physics were respected exactly. What this extension defines is a function that maps [0, infinity) input light RGB to the [0, 1] output encoding color gamut (most commonly sRGB). In practice, this function has a much larger effect on the appearance of the output than the differences in renderer approximations.
+
+Supporting this extension is only appropriate for rendering a single glTF scene; since it defines a global output mapping it is impossible to respect conflicting mappings for two glTFs in the same scene. Likewise, if a shop wanted a consistent “look” across  their portfolio, they might be better served by picking a consistent mapping function and requiring their artists to design with this mapping in mind, rather than displaying different products with unique parameters. 
+
+This mapping function is split into two parts: the linear multiplier, exposure, and the nonlinear look-up tables (LUT). Exposure represents what the human pupil does to compensate for scenes with different average light levels, but here we will define it in terms from the photography world, as they have already created standard physical units to represent the camera’s version of the pupil. 
```

Comment:
Reading Annex A it seems to me that that extension is modelling a "physical camera", rather than the human eye.
My suggestion would be to move that behavior (ie `exposure` out of `image formation` into it's own extension where the sole focus is to extend the current camera object.

Reply:
>I would contend that the the human eye and a physical camera are very similar sensors

Ok, here we disagree, to me a camera is substantially different.

I will try to explain what I mean:
A camera has both shutter(speed) and ISO as well as the option of changing lenses.
Where a camera will typically open/close the shutter (or by some other means let in light for a set time period) they photoreceptive cells in the retina will continuously send information that is interpreted by the visual cortex.

The effect of having a light sensitive medium and long shutterspeed can never be replicated by human vision.
Below roughly 10 lumen human vision becomes monochromatic (black & white)

Similarly for the opposite. When a large amount of light is captured on the camera medium colors will often go to white.
This will not happen in human vision.

Reply:
> Below roughly 10 lumen human vision becomes monochromatic (black & white)

Scotopic vision is even more complicated than this as the rods and cones use the same pathways, which leads to interesting effects in low light that are not just black and white. Here's an attempt at replicating scotopic vision based on a few research papers: https://github.com/google/filament/pull/4559

Reply:
> Ok, here we disagree, to me a camera is substantially different.

What does the outlined proposal represent if we are dividing into “camera photography-like” versus “human visual system-like”?

Reply:
> Scotopic vision is even more complicated than this as the rods and cones use the same pathways, which leads to interesting effects in low light that are not just black and white.

Sure, there are light conditions that leads to mesopic vision - a combination of rods and cones.

**That does not change my argument that a camera is fundamentally different to the human eye, for instance:**

- A camera shutter can stay open for any duration, continuously exposing the film medium to photons until the shutter closes.

- The camera aperture can be set to any size in any light conditions, exposing the film medium to any number of photons.
 (Whereas the eye pupil is mostly there to protect the photoreceptor cells from too much energy, ie light)
 
- In a camera the medium being exposed to light is not always the same, it may vary greatly depending on type of film or what type of digital sensor is being used.